presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/TypeConverter.java

```diff
@@ -87,6 +91,9 @@
 
 public final class TypeConverter
 {
+    public static final String ORC_ICEBERG_ID_KEY = "iceberg.id";
+    public static final String ORC_ICEBERG_REQUIRED_KEY = "iceberg.required";
```

Comment:
This appears to be unused. Why is it stored in the file? It seems like this would only need to exist in the table metadata, not at read time for the file.

Reply:
It's part of the updated spec: https://github.com/apache/incubator-iceberg/pull/227/files#diff-116b0afb33e66021cf000896fde94817R509

I think we can think about whether we want to use it at read time later